objcgen/embedder.cs

```diff
@@ -663,7 +663,8 @@ public int Compile ()
 							mmp.Append ("-p "); // generate a plist
 							mmp.Append ($"--target-framework {GetTargetFramework ()} ");
 							string extensionFlag = Extension ? "-fapplication-extension" : "";
-							mmp.Append ($"--link_flags={extensionFlag} -force_load {Utils.Quote (Path.GetFullPath (sdk_output_file))} ");
+							string forceLoad = $"-force_load {Utils.Quote (Path.GetFullPath (sdk_output_file))}";
+							mmp.Append ($"--link_flags={Utils.Quote (extensionFlag + " " + forceLoad)}");
```

Comment:
I'm not sure you can double-quote - even less since you're giving it a " " that will be quoted.

We know `extensionFlag` does not need quoting (it's constant) so I expected something like:

```
var extensionFlag = Extension ? "-fapplication-extension " : String.Empty;
mmp.Append ($"--link_flags=\"{extensionFlag}{Utils.Quote (forceLoad)}");
```